js-exercises/negative-index/negativeIndex.js

```diff
@@ -0,0 +1,30 @@
+function negativeIndex(originalObject) {
+  if (!Array.isArray(originalObject)) {
+    throw new TypeError('Only arrays are supported');
+  }
+  return new Proxy(originalObject, {
+    get(target, property) {
+      let returnObj;
+      if (property in target) {
+        returnObj = target[property];
+      } else if (typeof property !== 'symbol') {
+        let propValue = Number.parseInt(property, 10);
```

Comment:
Use of 'let' keyword is good, since we are not re-assigning the value you can use 'const' keyword also.

Reply:
@NaveenMN23 I have actually re-used the variable propValue on lines 13 and 14. I hope that is a satisfactory reason for using 'let' instead of 'const'.